chat-server/package.json

```diff
@@ -22,16 +22,16 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@azure/openai": "^1.0.0-beta.2",
+    "chat-core": "file:../chat-core",
```

Comment:
this is easier than i expected! thought would need some npm link or smthn like that